numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1770,28 +1788,6 @@ unpack_bits(PyObject *input, int axis, PyObject *count_obj)
         goto fail;
     }
 
-    /* setup lookup table under GIL, big endian 0..256 as bytes */
-    if (unpack_init == 0) {
```

Comment:
how does the removal of the lookup table impact performance?
As I recall it was significant when I added it.